### DISCUSSION TITLE
trying to install using docker-compose - getting curl 403 error:

### DISCUSSION THREAD

**Discussion by kaplandani** [mediagis/nominatim-docker#431] [Q&A] [answered]
ominatim  | + tailpid=0
nominatim  | + replicationpid=0
nominatim  | + trap stopServices SIGTERM TERM INT
nominatim  | + /app/config.sh
nominatim  | + id nominatim
nominatim  | + useradd -m -p 'Aa12341234!!' nominatim
nominatim  | + IMPORT_FINISHED=/var/lib/postgresql/14/main/import-finished
nominatim  | + '[' '!' -f /var/lib/postgresql/14/main/import-finished ']'
nominatim  | + /app/init.sh
nominatim  | + OSMFILE=/nominatim/data.osm.pbf
nominatim  | + CURL='curl -L -A "mediagis/nominatim-docker:4.2.1" --fail-with-body'
nominatim  | + '[' true = true ']'
nominatim  | Downloading Wikipedia importance dump
nominatim  | + echo 'Downloading Wikipedia importance dump'
nominatim  | + curl -L -A '"mediagis/nominatim-docker:4.2.1"' --fail-with-body https://nominatim.org/data/wikimedia-importance.sql.gz -o /nominatim/wikimedia-importance.sql.gz
nominatim  |   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
nominatim  |                                  Dload  Upload   Total   Spent    Left  Speed
100   153  100   153    0     0    339      0 --:--:-- --:--:-- --:--:--   338
**nominatim  | curl: (22) The requested URL returned error: 403**

Testing the URL with Curl returns 403, wget works....
what is the real issue here, how can I resolve this ?

**Comment by leonardehrenfried** [comment, marked as answer]
It's probably because of this: https://github.com/mediagis/nominatim-docker/issues/416

**Comment by kaplandani** [comment]
wget works, curl doesn't. anyway around it ?

**Reply by leonardehrenfried** [reply]
https://github.com/mediagis/nominatim-docker/issues/420#issuecomment-1406171553

**Reply by kaplandani** [reply]
this helped, I think - I've used a chrome agent and the container start running but for some reason the logs are gibberish. so I can't tell what's going on.
